### PR TITLE
CHANGED TITLE AND MAIN HEADING

### PR DESCRIPTION
**Pull request by Vishwanathanselvamoorthy**
CHANGED TITLE AND MAIN HEADING

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <html lang="en">
 
 <head>
-  <title>FRESH CAREERS</title>
+  <title>FRESH CAREERS | MY CAREER PAGE</title>
```
Comment (lavanyaselvam09) [inline src/index.html:6]: good work vishwa

File: src/index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
     <div class="description">
 
-      <h1>About Freshh</h1>
+      <h1>ABOUT FRESH</h1>
```
Comment (lavanyaselvam09) [inline src/index.html:33]: perfect

**Review by lavanyaselvam09 — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Vishwanathanselvamoorthy) [reply src/index.html:6]: thank you sir

File: src/index.html (hunk 2 above)
Comment (Vishwanathanselvamoorthy) [reply src/index.html:33]: thank you

**Review by lavanyaselvam09 — APPROVED** [review]

**Pull request merged**